### PR TITLE
Header feature branch

### PR DESCRIPTION
**Pull request by DammyShittu**
Created;
1) Mobile View Header and
2) Mobile View Headline

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,62 @@
...
+
+Figma
+
+# 👤 Author
+
+GitHub: (@dammyShittu)
+
+Twitter: (@aded_shittu)
+
+LinkedIn: (linkedin.com/in/adedamola-shittu-3ab465172/)
```
Comment (hemant-soni-vst-au4) [inline README.md:48]: - [ ] Kindly make the links active. Please follow this [link](https://github.com/microverseinc/readme-template) for `README` template.

File: README.md [hunk 2]
```diff
@@ -1 +1,62 @@
...
+![Image of the header](./img/header.png)
+
+## Headline Image
+
+![Image of the headline](./img/headline.png)
+
+
+# Live Website
+
+(https://dammyshittu.github.io/Portfolio-Project/)
```
Comment (hemant-soni-vst-au4) [inline README.md:22]: - [ ] Please try to follow template guidelines to make the `live link`.  we can use `[Live Demo Link](https://dammyshittu.github.io/Portfolio-Project/)` to make the readme file more presentable.

File: index.html [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+ <link rel="stylesheet" href="style.css">
+</head>
+<body>
+
+ <!-- Header Section -->
+
+ <header>
+  <div id="page_logo">
+   <span>Dammy Shittu</span>
+  </div>
```
Comment (hemant-soni-vst-au4) [inline index.html:17]: - [ ] Please try to add logo in `<a>` tag. It will help the user to come back to the home page.

File: index.html [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+
+ <!-- End of Header Section -->
+
+ <!-- Main Section -->
+ <main >
+  <section class="container">
+   <div id="circles">
+    <img src="./img/ellipse-1.svg" class="circle-1" alt="Circle 1">
+    <img src="./img/ellipse-2.svg" class="circle-2" alt="Circle 2">
+   </div>
```
Comment (hemant-soni-vst-au4) [inline index.html:33]: - [ ] Please try to add a proper background image as followed in the `Figma` design. The decorations are always background image, not `img` tags.

File: index.html [hunk 5]
```diff
@@ -0,0 +1,58 @@
...
+  
+   <div id="portfolio-intro">
+    <h1>Adedamola Shittu</h1>
+    <p>Software Developer</p>
```
Comment (hemant-soni-vst-au4) [inline index.html:37]: - [ ] Kindly try to add the `hx` tag for subtitle in place of the `<p>` tag. As per Html conventions, we have to use any other heading tag for subtitle.

File: index.html [hunk 6]
```diff
@@ -0,0 +1,58 @@
...
+    <section class="talent">
+     <div class="can-do language">
+      <h2>Language</h2>
+     </div>
+     <div class="can-do framework">
+      <h2>Framework</h2>
+     </div>
+     <div class="can-do skills">
+      <h2>Skills</h2>
+     </div>
```
Comment (hemant-soni-vst-au4) [inline index.html:52]: - [ ] Please try to add `<ul>` and  `<li>` tag to group the list. And use the `<a>` tag for each icon.

**Review by hemant-soni-vst-au4 — CHANGES_REQUESTED** [review]
Hi @DammyShittu, 

Good job so far! 👍 
There are some issues that you still need to work on to go to the next project but you are almost there! 

Alt Text
## Required Changes ♻️
_Check the comments under the review._

- [ ] Use of `<a>` tag for logo
- [ ] Change for background image
- [ ]  Add subheading in other heading tag
- [ ] Use `<ul>` tag for icons
- [ ] Make links active in `README` file



Cheers,
Hemant Soni
Cheers and happy coding 😃

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. You can mention me (@hemant-soni-vst-au4) so I can see your comments and respond.


**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by carlos-ssh — APPROVED** [review]
Hi @DammyShittu , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**